test/contrib/emscripten/test_emscripten.py

```diff
@@ -682,11 +686,10 @@ def test_streaming_not_ready_in_browser(
     )
 
 
-@install_urllib3_wheel()
 def test_requests_with_micropip(
     selenium_coverage: typing.Any, testserver_http: PyodideServerInfo
 ) -> None:
-    # this can't be @run_in_pyodide because of the async code
+    # this can't be @run_in_pyodide(packages=['urllib3']) because of the async code
```

Comment:
Unrelated to this PR, but it should work just fine to do:
```py
@run_in_pyodide
async def test_something(...):
    await some_async_call()
```
See for instace: https://github.com/pyodide/pyodide/blob/main/src/tests/test_browser_apis.py

Reply:
yeah, I just didn't want to have to add an asyncio test plugin just for this single test